<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="import_config">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导入配置文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_project">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成项目文件</string>
    </property>
   </widget>
   <widget class="QComboBox" name="current_node">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>390</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="node_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>节点名称:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tcp_protocol">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
   <widget class="QLabel" name="current_node_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="transform_protocol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>传输协议：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="udp_protocol">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_type">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>数据类型：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="war_message">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>战术消息</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="voice_message">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>语音</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="image_message">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>490</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>图像</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="video_message">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>视频</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_amount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>数据量：</string>
    </property>
   </widget>
   <widget class="QWidget" name="network_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>361</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>实际坐标X：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>实际坐标Y：</string>
    </property>
   </widget>
   <widget class="QLabel" name="XPosition_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="YPosition_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>520</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="amountUnit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>bytes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process_result_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>仿真结果处理</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dataAmountEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>520</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="alternate_paths_box">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="transmission">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>传输介质:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="medium_653">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>G.653</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="medium_652">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>G.652</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="medium_655">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>G.655</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
